frontend/src/Sections.css

```
@@ -0,0 +1,434 @@
+  border-left: 3px solid var(--color-primary-light);
+}
+
+.user-item:hover {
+  background: rgba(37, 99, 235, 0.05);
+  transform: translateX(4px);
+}
+
+.leader-item {
+  background: linear-gradient(135deg, rgba(147, 51, 234, 0.1), rgba(196, 181, 253, 0.1));
+  border-left-color: var(--color-accent);
+  border: 1px solid rgba(147, 51, 234, 0.2);
```

Stylelint (17.4.0)

[error] Unexpected shorthand "border" after "border-left-color" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)